### ISSUE TITLE
[1.16] Tungsten Ore Exploit with Immersive Engineering

### ISSUE THREAD

**Issue by Flexico** [Qmunity/BluePower#579]
When I put a Tungsten ore into an Arc Furnace from IE, I get two tungsten ingots instead of two nuggets! I told the IE people about it and they suggested that you register tungsten ore as "forge:nugget_ores/tungsten" instead of "forge:ores/tungsten" :)

**Comment by MoreThanHidden** [comment]
Fair enough, I wasn't sure if I was going to keep it this way just wanted to minimize the amount of tungsten being created until it had a use case.

**Comment by Flexico** [comment]
It is pretty doggone rare on top of only getting a nugget per ore! Considering that, I think Tungsten tools should come with a serious buff, like infinite durability or something.

**Comment by Flexico** [comment]
This has been fixed in 1.18 since tungsten ore now smelts to an ingot instead of a nugget!
